intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -8,15 +10,22 @@ editor_options:
   chunk_output_type: inline
 ---
 
-**CCDL 2020**
 
-## Objective for this notebook analysis: 
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Use the tidyverse package to read and write data frames
+- Use dplyr pipes to manipulate data frames
+- Implement and use functions in the tidyverse package to wrangle data (i.e. filter, mutate, arrange, join)
```

Comment:
I switched the order here, and tried to a note that pipes are more general than just for data frames. 
I also changed the package name for pipes to magrittr; we should probably make that change in the notebook as well on line 202

```suggestion
- Implement and use functions in the tidyverse package to wrangle data (i.e. filter, mutate, arrange, join)
- Use `magrittr` pipes (`%>%`) to combine multiple operations 
```

Reply:
Github automatically marked this as resolved, but I unresolved it to highlight the change on line 202 (or thereabouts) where we have a header `## dplyr pipes` which should be changed to `## magrittr pipes`

Reply:
Okay should be all set.